Traceback (most recent call last):
  File "/data/mluo/miniconda3/envs/ray/lib/python3.6/site-packages/ray/tune/trial_runner.py", line 436, in _process_trial
    result = self.trial_executor.fetch_result(trial)
  File "/data/mluo/miniconda3/envs/ray/lib/python3.6/site-packages/ray/tune/ray_trial_executor.py", line 323, in fetch_result
    result = ray.get(trial_future[0])
  File "/data/mluo/miniconda3/envs/ray/lib/python3.6/site-packages/ray/worker.py", line 2195, in get
    raise value
ray.exceptions.RayTaskError: [36mray_IMPALA:train()[39m (pid=1818, host=c80)
  File "/data/mluo/miniconda3/envs/ray/lib/python3.6/site-packages/ray/rllib/agents/trainer_template.py", line 87, in __init__
    Trainer.__init__(self, config, env, logger_creator)
  File "/data/mluo/miniconda3/envs/ray/lib/python3.6/site-packages/ray/rllib/agents/trainer.py", line 324, in __init__
    Trainable.__init__(self, config, logger_creator)
  File "/data/mluo/miniconda3/envs/ray/lib/python3.6/site-packages/ray/tune/trainable.py", line 87, in __init__
    self._setup(copy.deepcopy(self.config))
  File "/data/mluo/miniconda3/envs/ray/lib/python3.6/site-packages/ray/rllib/agents/trainer.py", line 439, in _setup
    self._init(self.config, self.env_creator)
  File "/data/mluo/miniconda3/envs/ray/lib/python3.6/site-packages/ray/rllib/agents/trainer_template.py", line 103, in _init
    self.workers = make_workers(self, env_creator, policy, config)
  File "/data/mluo/miniconda3/envs/ray/lib/python3.6/site-packages/ray/rllib/agents/impala/impala.py", line 101, in defer_make_workers
    return trainer._make_workers(env_creator, policy, config, 0)
  File "/data/mluo/miniconda3/envs/ray/lib/python3.6/site-packages/ray/rllib/agents/trainer.py", line 484, in _make_workers
    logdir=self.logdir)
  File "/data/mluo/miniconda3/envs/ray/lib/python3.6/site-packages/ray/rllib/evaluation/worker_set.py", line 64, in __init__
    RolloutWorker, env_creator, policy, 0, self._local_config)
  File "/data/mluo/miniconda3/envs/ray/lib/python3.6/site-packages/ray/rllib/evaluation/worker_set.py", line 214, in _make_worker
    _fake_sampler=config.get("_fake_sampler", False))
  File "/data/mluo/miniconda3/envs/ray/lib/python3.6/site-packages/ray/rllib/evaluation/rollout_worker.py", line 313, in __init__
    self._build_policy_map(policy_dict, policy_config)
  File "/data/mluo/miniconda3/envs/ray/lib/python3.6/site-packages/ray/rllib/evaluation/rollout_worker.py", line 717, in _build_policy_map
    policy_map[name] = cls(obs_space, act_space, merged_conf)
  File "/data/mluo/miniconda3/envs/ray/lib/python3.6/site-packages/ray/rllib/policy/tf_policy_template.py", line 144, in __init__
    obs_include_prev_action_reward=obs_include_prev_action_reward)
  File "/data/mluo/miniconda3/envs/ray/lib/python3.6/site-packages/ray/rllib/policy/dynamic_tf_policy.py", line 174, in __init__
    action_prob = self.action_dist.sampled_action_prob()
  File "/data/mluo/miniconda3/envs/ray/lib/python3.6/site-packages/ray/rllib/models/action_dist.py", line 70, in sampled_action_prob
    return tf.exp(self.logp(self.sample_op))
  File "/data/mluo/miniconda3/envs/ray/lib/python3.6/site-packages/ray/rllib/models/action_dist.py", line 176, in logp
    tf.reduce_sum(self.log_std, reduction_indices=[1])) + tf.reduce_sum(2. * (np.log(2.) - y - tf.nn.softplus(-2. * y)), reduction_indices=[1])
NameError: name 'y' is not defined

